modules/exploits/windows/http/flexdotnetcms_upload_exec.rb

```diff
@@ -304,7 +304,7 @@ def cleanup
       unless res && res.code == 200 && res.body.exclude?(file)
         print_error("Failed to delete #{file}.")
         print_warning("Manual cleanup of #{file} is required.")
-        return
+        next
```

Comment:
This doesn't look right to me. `next` is normally used in loops, yet I don't see a loop being used here in this code. Am I missing something?

Reply:
This is being called in a loop, and looked like the module developer chose `return` over `next`, potentially due to their familiarity with JavaScript

Reply:
If it's useful knowledge, the code viewer on Github PRs has button for expanding the surrounding code which can be useful for getting additional context on code changes:

![image](https://user-images.githubusercontent.com/60357436/109136644-3e39da80-7750-11eb-98eb-4e39c843053c.png)

Full code with surrounding loop:

![image](https://user-images.githubusercontent.com/60357436/109136519-1e0a1b80-7750-11eb-9e28-36136021aca5.png)

Reply:
And example snippet using `next`:
```
2.7.2 :001 > [1,2,3].each { |x| next }
 => [1, 2, 3] 
```

Using returns:

```
2.7.2 :002 > [1,2,3].each { |x| return }
Traceback (most recent call last):
	...
(irb):2:in `block in <main>': unexpected return (LocalJumpError)
```

Reply:
Ah thanks I tried expanding this surrounding code but failed to see the loop there. Marking this as resolved, this was my mistake.